assets/icon/Discord-Logo-Color.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="71" height="55" viewBox="0 0 71 55" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please rename the icon to `discord`.svg ?

Reply:
Done 😉